### ISSUE TITLE
Document donated resources that the Kata community is using

### ISSUE THREAD

**Issue by ildikov** [kata-containers/kata-containers#9958]
**Which feature do you think can be improved?**

Documentation

**How can it be improved?**

The Kata community is using donated hardware and software resources for the contributors day-to-day upstream work. As these resources are currently not documented, it is hard to follow what is donated, who the main contacts are, what the timeframe of the donation is, etc.

This issue can be addressed by creating a new page in the Kata documentation and collect the list of services that is being donated to the community. It is recommended to document and maintain the following information, where applicable:

- Name of the 'open source program' Kata is participating in
  - Link to any publicly available information about the program
- Donating organization
- Use of the donated resource
- Available credits
- Timeframe of the donation
- Main contacts from Kata
- Main contacts from the donating organization

Remaining questions:
- Where in the Kata documentation would this information fit the best?
- What programs does Kata participate in?
- Which organizations are donating Kata resources outside of open source programs?

**Comment by sprt** [comment]
@ildikov I started a table in your original post to list out the different programs - I suggest we all contribute to it and decide where to place it.

**Comment by ildikov** [comment]
@sprt Perfect, thank you!
